frontends/web/src/components/rates/rates.tsx

```diff
@@ -34,6 +34,25 @@ export interface SharedProps {
 }
 
 export const currencies: Fiat[] = ['AUD', 'BRL', 'CAD', 'CHF', 'CNY', 'EUR', 'GBP', 'HKD', 'ILS', 'JPY', 'KRW', 'NOK', 'RUB', 'SEK', 'SGD', 'USD', 'BTC'];
+export const currenciesWithDisplayName: FiatWithDisplayName[] = [
+  { currency: 'AUD', displayName: 'Australian dollar' },
+  { currency: 'BRL', displayName: 'Brazilian real' },
+  { currency: 'CAD', displayName: 'Canadian dollar' },
+  { currency: 'CHF', displayName: 'Swiss franc' },
+  { currency: 'CNY', displayName: 'Chinese yuan' },
+  { currency: 'EUR', displayName: 'Euro' },
+  { currency: 'GBP', displayName: 'British pound' },
+  { currency: 'HKD', displayName: 'Hong Kong dollar' },
+  { currency: 'ILS', displayName: 'Israeli new shekel' },
+  { currency: 'JPY', displayName: 'Japanese yen' },
+  { currency: 'KRW', displayName: 'South Korean won' },
+  { currency: 'NOK', displayName: 'Norwegian krone' },
+  { currency: 'RUB', displayName: 'Russian ruble' },
+  { currency: 'SEK', displayName: 'Swedish krona' },
+  { currency: 'SGD', displayName: 'Singapore dollar' },
+  { currency: 'USD', displayName: 'United States dollar' },
+  { currency: 'BTC', displayName: 'Bitcoin' }
```

Comment:
I thought about if we should display the currency name in the native language. i.e. "Svensk krona" instead of "Swedish krona", but that does't really work for "Swiss france" with our 4 official languages 🤦 

so I guess better to have everything in English.

Reply:
Please note we have new Fiat currencies on master, maybe merge master->staging-revamp-settings first?

https://github.com/digitalbitbox/bitbox-wallet-app/pull/2141